notebooks/118-optimize-preprocessing/118-optimize-preprocessing.ipynb

```diff
@@ -0,0 +1,533 @@
+{
```

Comment:
<p>can we split this into 2 parts:</p><ul><li>convert model with preprocessing using mo</li><li>convert model without preprocessing in mo and add preprocessing using preprocessing API</li></ul><p>and then compare these 2 steps (and see that these steps lead to the sample performance/accuracy of results)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/811/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
tensorflow preprocessing is fine :)